targets/TARGET_TOSHIBA/TARGET_TMPM066/us_ticker.c

```diff
@@ -98,5 +110,5 @@ void us_ticker_disable_interrupt(void)
 
 void us_ticker_clear_interrupt(void)
 {
-    //no flags to clear
+    NVIC_ClearPendingIRQ(INT16A1_IRQn);
```

Comment:
I believe this should be part of fire interrupt now, clear can stay empty as it was?